git-server/README.md

```diff
@@ -5,3 +5,41 @@
 # Running
 
     $ radicle-git-server --root ~/.radicle
+
+# Git Hooks
+
+Git [hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) are used by the git http backend to manage requests made to a repository, such as a `push` action. Hooks are executable files that accept standard input, perform some action and return an exit status back to the sender of the request, either successfully completing the request or declining.
+
+This crate includes a `bin` folder that contains hooks used by the `radicle-git-server` for authorizing requests and performing other tasks.
+
+In order to use these hooks, the binaries in the `bin` folder must be built using `cargo build --bin pre-receive --feature hooks` and `cargo build --bin post-receive --feature hooks` (`--release` for production) and moving the binary file to the git server's project root `git/hooks/` (e.g. `~/.radicle/git/hooks/`) folder. These files will automatically be called by the http-backend.
+
+## Authorizing Signed Push Certificates in `pre-receive` Hook
+
+The `pre-receive` hook is the first hook invoked by the git http-backend when a `receive-pack` event is triggered from a `git push` client request.
+
+The `pre-receive` hook is used to verify the git signed push certificates from the sender, e.g. `git push --signed ...`. Unless the `git-server` is run with the `--allow-unauthorized-keys (-u)` flag, any unsigned git push will be denied by the `pre-receive` hook.
```

Comment:
I just noticed we have a short option for this (`-u`), I would suggest that we remove that, as it's too easy to copy paste from somewhere without understanding what it does.

Reply:
This line should also be updated ;)

Reply:
1b0cae5 fixes.